modules/claims_api/spec/fixtures/v2/veterans/disability_compensation/form_526_json_api.json

```diff
@@ -198,7 +198,7 @@
                 "retiredStatus": "RETIRED",
```

Comment:
Do we need to make changes to request.json, or example.json? Do we need to run `rake rswag:claims_api:build`?

Reply:
Updated example.json and added new swagger file.